docker/secondary-analysis-python/tools/submit/tests/test_create_analysis_json.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import os
+import sys
+import json
+
+#pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+#sys.path.insert(0, pkg_root)
+
+import submit.create_analysis_json as aj
```

Comment:
I think using a more descriptive name, like `analysis_json` instead of `aj` would be more readable